app/views/assignments/_read.html.erb

```diff
@@ -172,4 +184,16 @@
     <% end %>
   <% end %>
 <% end %>
+  <% unless @grouping.nil? %>
```

Comment:
Okay, so in terms of positioning, move this to above the "Submissions" section (right under the if condition on line 115)